WORKSPACE

```diff
@@ -10,11 +10,11 @@ workspace(name = "cilium")
 ENVOY_PROJECT = "envoyproxy"
 ENVOY_REPO = "envoy"
 
-# https://github.com/envoyproxy/envoy/tree/v1.21.3
+# https://github.com/envoyproxy/envoy/tree/v1.21.5
```

Comment:
1.21.5 is not released yet I reckon, but I understand that you are using commit hash from latest release/1.21 branch :heavy_check_mark:

Reply:
ah got it, this is to correct my mistake in previous bump :face_exhaling:

Reply:
> 1.21.5 is not released yet I reckon, but I understand that you are using commit hash from latest release/1.21 branch ✔️

1.21.5 was the last release available since 5 weeks ago, also our `ENVOY_VERSION` was already at `1.21.5`...

Reply:
Ref: https://github.com/envoyproxy/envoy/commits/release/v1.21